conans/test/unittests/model/settings_test.py

```diff
@@ -100,6 +100,9 @@ def test_get_safe(self):
         self.assertEqual(settings.os, "Windows")
         self.assertEqual(settings.get_safe("compiler.version"), None)
         self.assertEqual(settings.get_safe("build_type"), None)
+        self.assertEqual("Release", settings.get_safe("build_type", "Release"))
+        self.assertEqual(False, settings.get_safe("build_type", False))
+        self.assertEqual("Windows", settings.get_safe("os", "Linux"))
```

Comment:
about assertEqual signature, the correct form is (expected, current). Yeah, it sounds like yoda conditions ...